### PR TITLE
Add Official Website and Community and Instagram get image

### PR DESCRIPTION
**Pull request by aofdev**

### CODE REVIEW

**Comment by transitive-bullshit** [comment]
Thanks!!

File: readme.md [hunk 1]
```diff
@@ -14,10 +14,11 @@
 - [Testing](#testing)
 - [Services](#services)
 - [Articles](#articles)
+- [Community](#community)
 
 
 ## Official
```
Comment (githubcopyrightdoctor) [inline readme.md:20]: İ
```githubcopyrightdoctor

File: readme.md [hunk 2]
```diff
@@ -90,6 +92,9 @@
 - [What's the difference between Chromium and Chrome?](https://www.howtogeek.com/202825/what%E2%80%99s-the-difference-between-chromium-and-chrome/)
 
 
+## Community
+- [Slack Channel](https://join.slack.com/t/puppeteer/shared_invite/enQtMzU4MjIyMDA5NTM4LTM1OTdkNDhlM2Y4ZGUzZDdjYjM5ZWZlZGFiZjc4MTkyYTVlYzIzYjU5NDIyNzgyMmFiNDFjN2UzNWU0N2ZhZDc)
+
 ## Contribute
 
 Contributions welcome! Please read the [contributing guideline](contributing.md) first.
```
Comment (githubcopyrightdoctor) [inline readme.md:100]:
```suggestion
 Contributions welcome! Please read the [contributing guideline](contributing.md) first.
```

Comment (githubcopyrightdoctor) [reply readme.md:100]: # @githubcopyrightdoctor

**Review by githubcopyrightdoctor — APPROVED** [review]

**Pull request merged**